src/mmw/apps/modeling/tasks.py

```diff
@@ -357,6 +357,10 @@ def to_gms_file(mapshed_data):
     """
     Given a dictionary of MapShed data, uses GWLF-E to convert it to a GMS file
     """
+
```

Comment:
Consider removing this blank line between the docstring and the actual code. The only time we have it is if the first line is a comment.